Tasks/ServiceFabricDeploy/task.json

```diff
@@ -15,9 +15,9 @@
         "DeploymentGroup"
     ],
     "version": {
-        "Major": 1,
```

Comment:
Does this break compat with current version, if not can we avoid bumping it?

Reply:
@RoopeshNair After we support default value for diffPackagePath, we can avoid bumping the vertion